conjure-undertow-processor/src/main/java/com/palantir/conjure/java/undertow/processor/data/ParameterTypeVisitors.java

```diff
@@ -0,0 +1,180 @@
+/*
+ * (c) Copyright 2022 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.conjure.java.undertow.processor.data;
+
+import com.palantir.conjure.java.undertow.processor.data.ParameterType.Cases;
+import com.palantir.conjure.java.undertow.processor.data.ParameterType.SafeLoggingAnnotation;
+import com.squareup.javapoet.CodeBlock;
+
+public final class ParameterTypeVisitors {
+
+    public enum UsesRequestContextVisitor implements Cases<Boolean> {
+        INSTANCE;
+
+        @Override
+        public Boolean body(CodeBlock _deserializerFactory, String _deserializerFieldName) {
+            return false;
```

Comment:
fwiw this isn't supported in jersey, but I think we do allow it in conjure-undertow